README.md

```diff
@@ -1,4 +1,67 @@
-## Armore
+<h1 align="center">Welcome to Armore server 👋</h1>
+<p>
+</p>
 
+> Armore App Backend. A distributed system built with Rust and Node
 
-![logo](architecture/Architecture.png)
+### 🏠 [Homepage](https://armore.dev/)
+
+## Run
+
+We provide you with a set of tools that can help you to automate the building and running process throughout makefiles and docker-compose files.
+
+```bash
+$ cd <server_folder>
+$ make build_and_run # This command builds and run the docker containers
```

Comment:
There is one piece of setup before they can run:

From the root of the project you have to run this since the .env is ignored
```
cp .env.save .env
```

Reply:
yay okey I'll add that one